testing/bdd/features/segmentation/duplicate_sequence_id+nun.feature

```diff
@@ -10,28 +10,50 @@ Feature: Discard document with already seen pair of sequence identifier and sequ
   # SPEC-CONFORMANCE: R107
   Scenario: Discard document
     Given an xml file <xml_file>
-    And it has sequence identifier <seq_id>
-    And it has sequence number <seq_n>
-    When a previous document exists with sequence identifier <prev_seq_id> and sequence number <prev_seq_n>
+    And it has sequence identifier <seq_id_1>
+    And it has sequence number <seq_n_1>
+    And the document is added to the sequence
+    When another document arrives
+    And it has sequence identifier <seq_id_2>
+    And it has sequence number <seq_n_2>
     Then the document is not added to the sequence
 
     Examples:
-    | seq_id | seq_n | prev_seq_id | prev_seq_n |  
-    | n      | 1     | n           | 1          |  
-    # Leading zeros and + not permitted for xs:positiveInteger so there's not much else we can test here.
+    | seq_id_1 | seq_n_1 | seq_id_2 | seq_n_2 |  
+    | n        | 1       | n        | 1       |  
+    | 1        | 1       | 1        | 1       |  
+
+
+  Scenario: Do not discard document
+    Given an xml file <xml_file>
+    And it has sequence identifier <seq_id_1>
+    And it has sequence number <seq_n_1>
+    And the document is added to the sequence
+    When another document arrives
+    And it has sequence identifier <seq_id_2>
+    And it has sequence number <seq_n_2>
+    Then the document is added to the sequence
+
+    Examples:
+    | seq_id_1 | seq_n_1 | seq_id_2 | seq_n_2 |  
+    | n        | 0       | n        | 1       |  
+    | 0        | 1       | 1        | 1       |  
+
 
   # SPEC-CONFORMANCE: R108
   Scenario: Availability time unchanged when discarding
     Given an xml file <xml_file>
-    And it has sequence identifier <seq_id>
-    And it has sequence number <seq_n>
-    When a previous document exists with sequence identifier <prev_seq_id> and sequence number <prev_seq_n> 
-    And the previous document has availability time <prev_avail>
-    Then the previous document has availability time <post_avail>
+    And it has sequence identifier <seq_id_1>
+    And it has sequence number <seq_n_1>
+    And the document is added to the sequence
+    And the document has availability time <avail_time_1>    
+    When a document arrives 
+    And it has sequence identifier <seq_id_2> 
+    And it has sequence number <seq_n_2> 
+    Then the previous document has availability time <avail_time_2>
```

Comment:
Surely avail_time_1 and avail_time_2 are always the same in this case? What is the case where they would differ?